package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@opendnd/core",
-  "version": "0.2.7",
+  "name": "@howlingsails-dnd-core",
```

Comment:
I'm change this for publishing for validation in high projects for now

Reply:
Not sure what this means but cannot merge this if it changes the package name.